packages/v4/src/content/design-guidelines/components/login-page/login-page.md

```diff
@@ -5,6 +5,8 @@ section: components
 
 The **login page** allows a user to gain access to an application by entering their username and password or by authenticating using a social media login. Related design guideline: [Login](/components/login-page/design-guidelines)
 
+The login page allows a user to gain access to an application by entering their username and password or by authenticating using a social media login.
```

Comment:
This line shouldn't have been added. Now it repeats twice (I think it overrides her changes)